admin/admin.php

```diff
@@ -927,14 +926,14 @@ public function settings_validation( $input ) {
 			// show author name
 			$output['standard_source'] = 'author_name';
 		} else {
-			$output['standard_source'] = isset( $input['standard_source'] ) ? esc_attr( $input['standard_source'] ) : 'author_name';
+			$output['standard_source'] = isset( $input['standard_source'] ) ? esc_attr( $input['standard_source'] ) : 'custom_text';
```

Comment:
**Suggestion:** Consider using a constant for the string 'custom_text' as it is used multiple times in the code. This will make the code more maintainable and less prone to errors. [maintainability]
```suggestion
   define('DEFAULT_STANDARD_SOURCE', 'custom_text');
   $output['standard_source'] = isset( $input['standard_source'] ) ? esc_attr( $input['standard_source'] ) : DEFAULT_STANDARD_SOURCE;
```